docs/operations/licensing.md

```diff
@@ -78,55 +78,72 @@ Ondat cluster by browsing the __Licence__ page as shown below:
 
 ![license](/images/docs/concepts/licensing.png)
 
-## Ondat GUI - Obtaining a Licence via the GUI 
-
-Contact Ondat Customer Success team via getstarted@ondat.io and provide the
-following information:
-- **First** and **last** name
-- **Company** **name** and **role** (optional for Personal licence, mandatory
-  for Commercial licence)
-- **Cluster ID**
-
-In return to the registration, you will receive a licence file with a content
-similar to this one:
-
+## Ondat CLI - Running the CLI
+
+In order to get the CLI running, we must launch it on the target Kubernetes
+cluster:
+
+```shell
+kubectl -n storageos create -f-<<END
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storageos-cli
+  namespace: storageos
+  labels:
+    app: storageos
+    run: cli
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: storageos-cli
+      run: cli
+  template:
+    metadata:
+      labels:
+        app: storageos-cli
+        run: cli
+    spec:
+      containers:
+      - command:
+        - /bin/sh
+        - -c
+        - "while true; do sleep 3600; done"
+        env:
+        - name: STORAGEOS_ENDPOINTS
+          value: http://storageos:5705
+        - name: STORAGEOS_USERNAME
+          value: storageos
+        - name: STORAGEOS_PASSWORD
+          value: storageos
+        image: storageos/cli:v2.5.0
+        name: cli
+END
 ```
-clusterCapacityGiB: 5120 
-clusterID: 164237eb-f88a-4bb8-a7cf-a23d468e07c0 
-customerName: storageos
-expiresAt: "2021-11-15T14:00:00Z" 
-features:
-- nfs
-kind: project
 
-------------- LICENCE SIGNATURE -------------
-KyjNleTcdmieZVLmZ/rg0SzdAM7I/CH0j22FIFJJSJaeB71OvQrTMtHGyL5TSFNMrEGbyh1HQlDgZb5A
-V1HyjBlS3LjoB/MoagulTxIlZh/R8eRXCOQ46qNZ8Yb7+dHLdCVXBnRqZT11hLqZsMqIeO1y9f5dw65H
-kvl6vWW7YIS9r655S25jMMU7brrGDQVdjvU7tSA74BrnzDFHu7/poopIuFqcxZc/NLrKp/akkvyZI5Ex
-1wH7D4onjVG2pgi30Kia+mjbI1B9pxQyRppQQ4hNXy4qBUUNMFh0menh0wHdQoM1VLU4Il22PrkeICV0
-NaalLsK/96bJov6tpbg96g==
-```
+Be sure to edit the environment variables appropriately for your target cluster, eg. the
+username/password for the administrative user. 
 
-## Ondat GUI - Applying a Licence via the Ondat GUI To apply a licence:
-1.  Browse the __Licence__ page of the Ondat GUI
-1.  Click the __Upgrade__ button, for the specific licence level you purchased
-1.  Paste the licence key into the pop-up window
-1.  To apply the license click __Upgrade__
+Once the pod is launched, we can retrieve it's unique identifier:
```

Comment:
Same here - should be "you can retrieve"

Reply:
Fixed! Thanks.